.ci/test.sh

```diff
@@ -173,3 +173,73 @@ matplotlib.use\(\"Agg\"\)\
     conda install -q -y -n $CONDA_ENV ipywidgets notebook
     jupyter nbconvert --ExecutePreprocessor.timeout=180 --to notebook --execute --inplace *.ipynb || exit -1  # run all notebooks
 fi
+
+if [[ $TASK == "r-pkg" ]]; then
+    #. set up R environment
+    R_LIB_PATH=~/Rlib
```

Comment:
Borrowed your setup [from RGF](https://github.com/RGF-team/rgf/blob/master/R-package/.R.travis.sh) and adapted it slightly to (hopefully) avoid needing to build `devtools` and all its dependencies. Thanks @StrikerRUS !